### PR TITLE
SIG UI Annual Report 2021

### LINKED ISSUE #6384: 2021 Annual Report: SIG UI
Chairs: @floreks @maciaszczykm @shu-mutou 
Liaison: @justaugustus

Actions for the chair/organizer of the community group:
- [ ] Consult your community group to complete draft of report
- [ ] If needed, consult with Steering Committee liaison on any private issues or concerns that arrise while completing report
- [ ] Submit PR to git.k8s.io/community repo with copy of report, assigning your Steering Committee liaison for review
- [ ] Steering Committee reviews report, and may make comments/ask questions
- [ ] Once all comments are addressed, Steering Committee will approve report to merge
- [ ] If needed, any follow up items may be brought to the "Chairs, Tech Leads, and Organizers" meeting in April

Once all the above items are complete, this issue may be `/close`'d

Key dates:
- Initial PR to communtiy repo should be opened by February 14, 2022
- PR should be reviewed and merged by March 1st, 2022

More detailed information on the annual reports process is available [here](https://git.k8s.io/community/committee-steering/governance/annual-reports.md).

Your group's report template is available here: https://git.k8s.io/community/sig-ui/annual-report-2021.md

If you have any questions or concerns about this process, you may reach the Steering Committee via the following methods:
- Slack/E-mail your liaison directly
- The public #steering-committee channel in Slack
- The public mailing list steering@kubernetes.io
- The private mailing list steering-private@kubernetes.io for any private or sensitive issues.

/sig ui
/committee steering

### PR DESCRIPTION
**Pull request by shu-mutou**
Annual report 2021 for SIG UI.

**Which issue(s) this PR fixes**:

Fixes #6384

### CODE REVIEW

File: sig-ui/annual-report-2021.md
Comment (maciaszczykm) [inline sig-ui/annual-report-2021.md:8]: /s/Almost/Almost all
s/to latest/to the latest versions

File: sig-ui/annual-report-2021.md
Comment (maciaszczykm) [inline sig-ui/annual-report-2021.md:9]: One thing that I think may be worth mentioning here is that we have significantly updated our config, scripts etc. This opens space for multiple improvements that were blocked before.

File: sig-ui/annual-report-2021.md
Comment (floreks) [inline sig-ui/annual-report-2021.md:9]: s/we are improving support visualizing various resource objects/we are improving support for various resource objects

File: sig-ui/annual-report-2021.md
Comment (floreks) [inline sig-ui/annual-report-2021.md:10]: s/highlight is we/highlight is that we

**Comment by shu-mutou** [comment]
the commits squashed. Thanks!

**Comment by shu-mutou** [comment]
@floreks @maciaszczykm PTAL again. Thanks!

**Comment by floreks** [comment]
lgtm

**Comment by shu-mutou** [comment]
@justaugustus Can we merge this ourselves? If not so, could you review this?

**Comment by justaugustus** [comment]
In my review queue!

File: sig-ui/annual-report-2021.md
Comment (maciaszczykm) [reply sig-ui/annual-report-2021.md:15]: > where are you tracking your work?

Mostly on GitHub right now.

> what large (or small) initiatives are you working on now?

Larger one that is a long term plan is to create API v2. Smaller one is to keep the project up to date with Kubernetes releases, fix bugs that appear and also introduce new features like support for new kinds of resources.

Comment (shu-mutou) [reply sig-ui/annual-report-2021.md]: Sorry for late response. Updated this section. Thanks!

**Comment by shu-mutou** [comment]
@parispittman PTAL. Thanks!

**Comment by shu-mutou** [comment]
@maciaszczykm @floreks Could you merge this?

**Comment by shu-mutou** [comment]
@parispittman PTAL.

**Review by justaugustus — APPROVED** [review]
Thanks for the work here, folks!
Given the annual report has [shipped](https://www.cncf.io/reports/kubernetes-annual-report-2021/) and includes content from this, let's merge.

Feel free to address any comments as follow-ups (at your leisure).
/lgtm
/approve
/hold cancel

**Pull request merged**
